lib/cocoapods/installer/xcode/target_validator.rb

```diff
@@ -34,6 +34,7 @@ def initialize(aggregate_targets, pod_targets)
         def validate!
           verify_no_duplicate_framework_and_library_names
           verify_no_static_framework_transitive_dependencies
+          verify_no_pods_used_with_multiple_swift_versions
```

Comment:
This needs to move here since calling `uses_swift?` must ensure the `file_accessors` are set.